packages/okta-react/README.md

```diff
@@ -351,77 +446,175 @@ class App extends Component {
 
-Performs a full-page redirect to Okta with optional request parameters.
+For more information on `additionalParams`, see the [`authService.redirect`](#authredirectadditionalparams) method below.
+
+#### `authService.logout(uri)`
+
+(async method) Terminates the user's session in Okta and clears all stored tokens. Accepts an optional `uri` parameter to push the user to after logout.
+
+#### `authService.redirect(additionalParams)`
+
+(async method) Performs a full-page redirect to Okta with optional request parameters.
 
 The `additionalParams` are mapped to Okta's [`/authorize` request parameters](https://developer.okta.com/docs/api/resources/oidc#authorize). This will override any existing [configuration](#configuration-options). As an example, if you have an Okta `sessionToken`, you can bypass the full-page redirect by passing in this token. This is recommended when using the [Okta Sign-In Widget](https://github.com/okta/okta-signin-widget). Simply pass in a `sessionToken` into the `redirect` method as follows:
 
 ```jsx
-auth.redirect({
+authService.redirect({
   sessionToken: '{sampleSessionToken}'
 });
 ```
 
 > Note: For information on obtaining a `sessionToken` using the [Okta Sign-In Widget](https://github.com/okta/okta-signin-widget), please see the [`renderEl()` example](https://github.com/okta/okta-signin-widget#rendereloptions-success-error).
 
-#### `auth.handleAuthentication()`
+#### `authService.handleAuthentication()`
+
+(async method) Parses tokens from the url and stores them.  Done when handling a redirect back to the application from the issuer.  See the `LoginCallback` component for an example of use.
+
+### `authService.setFromUri(uri)`
+
+(synchronous method) Store the current URL state before a redirect occurs.  Called internally by `authService.login`.  If a relative path is passed it will be converted to an absolute URI before storage.
+
+#### `authService.getFromUri()`
+
+(synchronous method) Returns the stored URI string stored by `setFromUri` and removes it from storage.  See the `LoginCallback` component for an example of use.  
+
+#### `authService.getTokenManager()`
+
+(async method) Resolves with the internal [TokenManager](https://github.com/okta/okta-auth-js#tokenmanager).
 
-Parses tokens from the url and stores them.
+#### `authService.updateAuthState()`
 
-#### `auth.setFromUri(uri, queryParams)`
+(async method) Triggers a re-evaluation of whether a user is considered authenticated.  Might be need to be triggered manually for users that pass overrides to `isAuthenticated` to the Security component or to the `Auth` constructor.  Does NOT perform a login, simply re-evaluates the current authenticated status. An 'authStateChange' event is emitted once the re-evaluation is complete. 
```

Comment:
I think again it is not `isAuthenticated`, but `onAuthRequired` where you might want to call this function. It is called  by us right before we call `onAuthRequired`, using either the value passed from the SecureRoute or window.location.origin if no value was provided. A developer could make their own call to this method to cause a redirect to a different path after login. https://github.com/okta/okta-oidc-js/pull/677/files#diff-8a51cbb87b5cfd7e092f0410797f00e9R205